components/ad/extensionAdBlockDetector/src/index.js

```diff
@@ -0,0 +1,48 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+class AdExtensionAdBlockDetector extends Component {
```

Comment:
The name AdExtensionAdBlockDetector seems a bit redundant to me maybe we could change it by **AdBlockerExtensionDetector**

Reply:
Changed 👍